recipes/pipes/all/conanfile.py

```diff
@@ -0,0 +1,38 @@
+from conans import ConanFile, tools
+import os
+
+class PipesConan(ConanFile):
+    name = "pipes"
+    description = "Pipelines for expressive code on collections in C++"
+    license = "MIT"
+    topics = ("pipes", "functional-programming")
+    homepage = "https://github.com/joboccara/pipes"
+    url = "https://github.com/conan-io/conan-center-index"
+    settings = "compiler"
+    no_copy_source = True
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def configure(self):
+        minimal_cpp_standard = "14"
```

Comment:
You'll need the full conan-io/conan#8002 from the error

Reply:
Just curious, what's the advantage of checking in `configure()` and raise error if not valid, compared to removing the unsupported compilers from `settings`?

Reply:
The CCI backend using the default `settings.yml` which means we have no control over the profiles it tries.

Deleting locally would cause package IDs to be generated that shouldn't.


#4261 might be interesting since its related